core/java/src/org/openda/blackbox/wrapper/BBStochModelInstance.java

```diff
@@ -93,14 +93,14 @@ public IObservationOperator getObservationOperator(){
 	 * For each constraintExchangeItemId for which there is a range validation constraint this map contains
 	 * the corresponding RangeValidationConstraint object.
 	 */
-	private Map<String, RangeValidationConstraint> constraintExchangeItemIdConstraintMap = new LinkedHashMap<String, RangeValidationConstraint>();
+	private Map<String, RangeValidationConstraint> constraintExchangeItemIdConstraintMap = new LinkedHashMap<>();
 	/**
 	 * All constraintExchangeItems that have been constructed so far are stored in this map.
 	 * This list is filled lazily, i.e. a constraintExchangeItem with constraint(s) is constructed (by wrapping the affected exchangeItem)
 	 * when it is first requested in method BBStochModelInstance.getExchangeItem. This is needed because
 	 * some (e.g. output) exchangeItems are not initialized when the model has not run yet.
 	 */
-	private Map<String, IPrevExchangeItem> constraintExchangeItems = new LinkedHashMap<String, IPrevExchangeItem>();
+	private Map<String, IExchangeItem> constraintExchangeItems;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/OpenDA-Association/OpenDA/pullRequest?prid=5102854)